src/utils/generation_utils.py

```diff
@@ -241,7 +246,23 @@ def generate_merchant_inventory(
 def generate_quest_arc(
     player: HumanCharacter, context: AgentContext
 ) -> List[QuestDescription]:
-    prompt = f"Please list 10 quests of increasing difficulty that {player.name} will go in to achieve their overall goal of {player.motivation}. They should fit the setting of the story. Please respond only with QUEST GOAL: <goal> QUEST LOCATION: <location name>"
+    prompt = (
+        f"Please list 10 quests of increasing difficulty that {player.name} will go in to achieve their overall "
+        f"goal of {player.motivation}. They should fit the setting of the story. Responses should only be in the "
+        f"form of: QUEST GOAL: <goal> QUEST LOCATION: <location name>\n"
+        f"Example (for a quest game for a dog):\n"
+        f"QUEST GOAL: find a treat QUEST LOCATION: Dog Park\n"
+        f"QUEST GOAL: make a friend QUEST LOCATION: Main Street\n"
+        f"QUEST GOAL: get bacon from the butcher QUEST LOCATION: Butcher Shop\n"
+        f"QUEST GOAL: learn a new trick QUEST LOCATION: Trainer's Office\n"
+        f"QUEST GOAL: fetch the newspaper QUEST LOCATION: Owner's House\n"
+        f"QUEST GOAL: prevent a robbery QUEST LOCATION: Owner's Store\n"
+        f"QUEST GOAL: impress the teacher during a show and tell QUEST LOCATION: Owner's Kid's School\n"
+        f"QUEST GOAL: steal some turkey QUEST LOCATION: Thanksgiving Dinner at Grandma's\n"
+        f"QUEST GOAL: get your nails trimmed QUEST LOCATION: Dog Wash\n"
+        f"QUEST GOAL: win an award QUEST LOCATION: Westminister Dog Show\n"
```

Comment:
Just curious, since you've done way more prompting than I have, I think... Does the flavor of the specific examples ever leak out into the generated contents?

Reply:
@dkolas i hope not, but it seems inevitable. i wasn't feeling particularly creative when writing it. I'd love for someone to go back and put a little pizzaz on it.